mods/hv/rules/pods.yaml

```
@@ -268,32 +268,34 @@ BROKER:
 		Range: 2c0
 	Health:
 		HP: 17500
-	GrantConditionOnDeploy:
-		PauseOnCondition: jammed
-		DeployedCondition: deployed
-		UndeployedCondition: undeployed
-		UndeployOnMove: true
-		UndeployOnPickup: true
-		Facing: 512
-		DeploySounds: broker_deploy_mincedbeats.ogg
-		UndeploySounds: broker_undeploy_mincedbeats.ogg
-		AllowedTerrainTypes: Clear, Road, Crater, Grass, Grass Ramp, Grass Pit, Ore, Mountain, Mountain Ramp, Rock, Ice, Snow, Snow Ramp, Sand, Sand Ramp, Stone, Tech, Dirt
-		Voice: Deploying
-	WithMakeAnimation:
-		Sequence: extend
-		BodyNames: deployed
 	WithFacingSpriteBody:
-		RequiresCondition: undeployed
-	WithSpriteBody@DEPLOYED:
-		RequiresCondition: !undeployed
-		Sequence: extended
-		Name: deployed
-	CashTrickler:
-		Interval: 50
-		Amount: 5
-		RequiresCondition: deployed
 	Voiced:
 		VoiceSet: BrokerVoice
+	GrantConditionOnAttack:
+		Condition: chargeup-stealing
+		RevokeDelay: 220
+		MaximumInstances: 11
+		RevokeOnNewTarget: true
+		RevokeAll: true
+		IsCyclic: true
+	Armament@PrepareStage1:
+		Weapon: BrokerPrepareStealingStage1
+		RequiresCondition: chargeup-stealing < 6
+	Armament@PrepareStage2:
+		Weapon: BrokerPrepareStealingStage2
+		RequiresCondition: chargeup-stealing > 5 && chargeup-stealing < 11
+	Armament@Steal:
+		Weapon: BrokerSteal
+		RequiresCondition: chargeup-stealing == 11
+	AttackFrontal:
+		FacingTolerance: 10
+		Voice: Attack
+	AutoTarget:
+		MinimumScanTimeInterval: 5
+		MaximumScanTimeInterval: 15
+	AutoTargetPriority@Default:
+		ValidTargets: Leechable
```

```suggestion
		ValidTargets: Defraudable
```
maybe to keep with financial terms?